schema/home/home_page_artist_module.js

```diff
@@ -0,0 +1,63 @@
+import {
+  has,
+  map,
+} from 'lodash';
+import Artist from '../artist';
+import gravity from '../../lib/loaders/gravity';
+import { NodeInterface } from '../object_identification';
+import { toGlobalId } from 'graphql-relay';
+import {
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+
+export const Results = {
+  suggested: ({ accessToken, userID }) => {
+    return gravity.with(accessToken)(`user/${userID}/suggested/similar/artists`).then(results => {
```

Comment:
This endpoint seems suspect to me. I mean it's fine, but the endpoint is like 3 years old (!), and it's just weird.

It requires a user to be logged in (so you need `accessToken`), but then it doesn't actually _use_ the `current_user` anywhere. Furthermore, it simply lets any logged-in user look up the suggested artists for any other user, just be specifying the id.

Basically, while this is fine, I think there's a case for making something more sensible in Gravity.